GVRf/Framework/framework/src/main/java/org/gearvrf/scene_objects/GVRVideoSceneObject.java

```diff
@@ -95,7 +96,7 @@ public GVRVideoSceneObject(final GVRContext gvrContext, GVRMesh mesh,
             @Override
             public void run() {
                 // Because texture.getId() is called, this needs to run in GL thread
-                mVideo = new GVRVideo(mediaPlayer, texture);
+                mVideo = new GVRVideo(gvrContext, mediaPlayer, texture);
                 gvrContext.registerDrawFrameListener(mVideo);
```

Comment:
The drawFrameListener needs to be unregistered at some point. Otherwise the GVRVideo instance will be still around for as long as the app runs.